### DISCUSSION TITLE
Sugestões de raças extras

### DISCUSSION THREAD

**Discussion by YuriAlessandro** [YuriAlessandro/gerador-ficha-tormenta20#256] [Ideias]
Sugiram raças extras para o projeto.

Coloquem sempre o link para o detalhamento da raça (ou se você criou, coloque o texto todo aqui mesmo) e sempre informe se você pediu autorização para o criador.

Vamos incluindo o que fomos achando legal.

**Comment by YuriAlessandro** [comment]
## Rhox - Os Homens rinocerontes

Os Rhox  são um povo  de homens  Rinocerontes que vive na grande savana e nas montanhas  sanguinárias. Seus tons de pele  vão do branco ao preto  passando  por tons de marrom.

Link: https://knightofhammer.blogspot.com/2021/03/rhox-os-homens-rinocerontes-para.html
Ainda não foi pedido autorização ao criador

**Comment by LeonardoCalazans** [comment]
## Orc

Não sei se vi direito, mas eu não achei o orc ou meio orc. Acho que expandir para o D&D 5e vai ser ainda mais top.

**Reply by YuriAlessandro** [reply]
Orc e Meio Orc não são classes oficiais de Tormenta 20, e se colocar só as de D&D 5e pode ser que não se encaixe bem com o sistema.

**Comment by MrLaw23** [comment]
Com a adição do livro "A lenda de Ghannor" ao Sistema T20 seria legal colocar as novas raças que vem no livro.
- Gigante
- Meio Elfo
- Hobgoblin
- Aberrante

Para classes não tem mt diferenças, só criaram uma classe só de magos
